README.md

```diff
@@ -1,2 +1,89 @@
 # steampipe-mod-tailscale-compliance
```

Comment:
This heading is not correct, can you please make it consistent with other Compliance mods?